src/Search.tsx

```diff
@@ -1,62 +1,29 @@
 import React from 'react';
-import { Hits, SortBy, Pagination } from 'react-instantsearch-dom'
-import { ProductHit } from './ProductHit';
-import { CustomRefinementList } from './CustomRefinementList';
-import { useTranslation } from './app-state';
-import { config } from './config';
+import { AlgoliaSearch } from './AlgoliaSearch';
+import { CoveoSearch } from './CoveoSearch';
 
-import './Search.scss';
+import { config } from './config';
 
 interface SearchParams  {
 }
 
 export const Search: React.FC<SearchParams> = () => {
-  const { t } = useTranslation();
-
-  const Hit = ({ hit }: any) => (
-    <div className="search__product">
-      <ProductHit hit={hit} />
-    </div>
-  );
-
-  const Facets = () => (
-    <div className="search__facets">
-      <div>
-        <h2 className="search__facetstitle">
-          {t('filter-by')}
-        </h2>
-        <SortBy
-          key="facets-SortBy"
-          defaultRefinement={config.algoliaIndexName}
-          items={[
-            { value: config.algoliaIndexName, label: t('featured') },
-            { value: `${config.algoliaIndexName}_price_asc`, label: t('price-asc') },
-            { value: `${config.algoliaIndexName}_price_desc`, label: t('price-desc') }
-          ]}
-        />
-      </div>
-      <CustomRefinementList key="facets-list-1" title={t('category')} attribute="categories" />
-      <CustomRefinementList key="facets-list-2" title={t('collection')} attribute="collections" />
-      <CustomRefinementList key="facets-list-3" title={t('brand')} attribute="brands" />
-    </div>
-  );
+  const renderSearch = () => {
+    switch(config.searchProvider) {
+      case 'coveo':
+        return (<CoveoSearch />)
+      case 'algolia':
+        return (<AlgoliaSearch />)
+      default:
+        return (
+          <p>Unsupported search provider</p>
```

Comment:
Please move this text to the messages.json for languages

Reply:
Moved that message to locales jsons.